src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -446,6 +451,7 @@ else if (block instanceof BlockSlab)
         }
         else
         {
+            //todo farmland doesn't have damage at all, sucker!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)